Validations Khadri 04112024 0240 pm

roohishaik128 opened the pull request:
Validations For Product & Stock Module

sgreddy123 committed:
StockModule

roohishaik128 committed:
Added Validation For Product & Stock Module

manvithagali52 commented on SQL_Scripts/scripts.sql:
```diff
@@ -0,0 +1,9 @@
+create table charger(product_brand varchar(50),product_price double,product_model varchar(50),arrived_date_time datetime);
+create table powerbank(product_brand varchar(50),product_price double,product_model varchar(50),arrived_date_time datetime);
+create table backcover(product_brand varchar(50),product_price double,product_model varchar(50),arrived_date_time datetime);
+create table headset(product_brand varchar(50),product_price double,product_model varchar(50),arrived_date_time datetime);
```
why we need sql scripts it is already in master branch

roohishaik128 replied:
I added Wrongly

manvithagali52 commented on src/main/java/com/khadri/jakarta/stock/dao/StockDao.java:
why you deleted this stockDao class

manvithagali52 commented on src/main/java/com/khadri/jakarta/stock/form/BackCoverForm.java:
```diff
@@ -0,0 +1,35 @@
+	}
+
+	public Date getArrivedDateTime() {
+		return arrivedDateTime;
```
why your are adding all this files

manvithagali52 commented on src/main/java/com/khadri/jakarta/stock/servlet/modify/StockModifyServlet.java:
```diff
@@ -0,0 +1,61 @@
+			}
+		}
+	}
+}
```
Add only your validation files why you are adding all files

sgreddy123 commented on src/main/java/com/khadri/jakarta/stock/form/BackCoverForm.java:
you need to add only your validation files.

sgreddy123 commented on src/main/java/com/khadri/jakarta/product/servlet/modify/ProductModifySearchPageServlet.java:
```diff
@@ -36,20 +36,38 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
+		sb.append("alert('Please enter a Product ID ..!');");
+		sb.append("return false;");
+		sb.append("}");
+		sb.append("if(isNaN(productIdComponent)) { ");
```
isNaN or isName?

roohishaik128 replied:
isNanN is method when productId given a alphabet It gives one alert

roohishaik128 committed:
Resolved Conflicts

Merge branch 'master' into KHADRI-04112024-0240-PM

# Conflicts:
#	src/main/java/com/khadri/jakarta/product/dao/ProductDao.java
#	src/main/java/com/khadri/jakarta/stock/dao/StockDao.java
#	src/main/java/com/khadri/jakarta/stock/servlet/add/StockAddPageServlet.java
#	src/main/java/com/khadri/jakarta/stock/servlet/add/StockAddServlet.java
#	src/main/java/com/khadri/jakarta/stock/servlet/delete/StockDeletePageServlet.java
#	src/main/java/com/khadri/jakarta/stock/servlet/modify/StockModifyPageServlet.java
#	src/main/java/com/khadri/jakarta/stock/servlet/modify/StockModifySearchPageServlet.java
#	src/main/java/com/khadri/jakarta/stock/servlet/modify/StockModifyServlet.java
#	src/main/java/com/khadri/jakarta/stock/servlet/view/StockViewServlet.java
#	src/main/java/com/khadri/jakarta/stock/servlet/viewall/StockViewAllServlet.java
#	src/main/webapp/WEB-INF/web.xml
#	src/main/webapp/bottom_left.html
